operations/service_runner.go

```diff
@@ -140,6 +142,7 @@ func startSystemCronJobs(ctx context.Context, env evergreen.Environment) {
 
 	const (
 		monitoringInterval      = time.Minute
+		taskPlanningInterval    = 15 * time.Second
```

Comment:
Shouldn't this match `20 * time.Second` on 267? Also, you could put the constant there and use it both places.